packages/grid/_modules_/grid/models/gridSlotsComponent.ts

```diff
@@ -46,7 +46,7 @@ export interface GridSlotsComponent extends GridIconSlotsComponent {
    */
   Footer?: React.ElementType<BaseComponentProps>;
   /**
-   * Header component rendered above the grid column header bar.
+   * Prefer using the `Toolbar` slot. You should never need to use this slot. TODO remove.
```

Comment:
I think you can leave the previous description and add that sentence after 🤔

Reply:
done